confidential-assets/src/confidentialTransfer.ts

```diff
@@ -188,42 +201,58 @@ export class ConfidentialTransfer {
     )
     const alpha4List = baseProofArray.slice(
       half + 1 + ConfidentialAmount.CHUNKS_COUNT,
-      half + 1 + ConfidentialAmount.CHUNKS_COUNT * 2,
+      (half + 1 + ConfidentialAmount.CHUNKS_COUNT) + ConfidentialAmount.CHUNKS_COUNT,
```

Comment:
This is a lot of manual deserialization work. I wonder if it can be automated more.